Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -349,23 +363,39 @@ open class KMConversationViewController: ALKConversationViewController {
 
     func updateAssigneeDetails() {
         conversationDetail.updatedAssigneeDetails(groupId: viewModel.channelKey, userId: viewModel.contactId) { contact, channel in
+            self.messageStatusAndFetchBotType()
             guard let alChannel = channel else {
                 print("Channel is nil in updatedAssigneeDetails")
                 return
             }
             self.customNavigationView.updateView(assignee: contact, channel: alChannel)
             self.assigneeUserId = contact?.userId
             self.hideInputBarIfAssignedToBot()
+            guard let contact = contact else {return}
+            self.isAwayMessageViewHidden = !contact.isInAwayMode
```

Comment:
We are already handling this in `messageStatusAndFetchBotType`, no?

Reply:
no its not handled there.

Reply:
ok